sdk/src/configs/tokens.ts

```diff
@@ -0,0 +1,1325 @@
+    },
+    {
+      name: "BNB",
+      symbol: "BNB",
+      decimals: 18,
+      priceDecimals: 3,
+      address: "0x110892Dd5fa73bE430c0ade694febD9a4CAc68Be",
+      isSynthetic: true,
+      coingeckoUrl: "https://www.coingecko.com/en/coins/binancecoin",
+    },
+    {
+      name: "Cardano",
+      symbol: "ADA",
+      decimals: 18,
+      priceDecimals: 5,
+      address: "0xE64dfFF37Fa6Fe969b792B4146cEe2774Ef6e1a1",
+      isSynthetic: true,
+      coingeckoUrl: "https://www.coingecko.com/en/coins/cardano",
+    },
+    {
+      name: "TRON",
+      symbol: "TRX",
+      decimals: 18,
+      priceDecimals: 5,
+      address: "0x0D1495527C255068F2f6feE31C85d326D0A76FE8",
+      isSynthetic: true,
+      coingeckoUrl: "https://www.coingecko.com/en/coins/tron",
+    },
+    {
+      name: "Polygon",
+      symbol: "MATIC",
+      decimals: 18,
+      priceDecimals: 4,
+      address: "0xadc4698B257F78187Fd675FBf591a09f4c975240",
+      isSynthetic: true,
+      coingeckoUrl: "https://www.coingecko.com/en/coins/polygon",
+    },
+    {
+      name: "Polkadot",
+      symbol: "DOT",
+      address: "0x65FFb5664a7B3377A5a27D9e59C72Fb1A5E94962",
+      decimals: 18,
+      priceDecimals: 4,
+      isSynthetic: true,
+      coingeckoUrl: "https://www.coingecko.com/en/coins/polkadot",
+    },
+    {
+      name: "Uniswap",
+      symbol: "UNI",
+      decimals: 18,
+      priceDecimals: 4,
+      address: "0xF62dC1d2452d0893735D22945Af53C290b158eAF",
+      isSynthetic: true,
+      coingeckoUrl: "https://www.coingecko.com/en/coins/uniswap",
+    },
+    {
+      name: "Dogecoin",
+      symbol: "DOGE",
+      decimals: 8,
+      priceDecimals: 5,
+      address: "0x2265F317eA5f47A684E5B26c50948617c945d986",
+      isSynthetic: true,
+      isShortable: true,
+      coingeckoUrl: "https://www.coingecko.com/en/coins/dogecoin",
+    },
+    {
+      name: "Chainlink",
+      symbol: "LINK",
+      decimals: 18,
+      priceDecimals: 3,
+      address: "0x6BD09E8D65AD5cc761DF62454452d4EC1545e647",
+      isSynthetic: true,
+      isShortable: true,
+      coingeckoUrl: "https://www.coingecko.com/en/coins/chainlink",
+    },
+    {
+      name: "XRP",
+      symbol: "XRP",
+      decimals: 6,
+      priceDecimals: 4,
+      address: "0xF1C2093383453831e8c90ecf809691123116dAaC",
+      isSynthetic: true,
+      imageUrl: "https://assets.coingecko.com/coins/images/44/small/xrp-symbol-white-128.png?1605778731",
+      coingeckoUrl: "https://www.coingecko.com/en/coins/xrp",
+    },
+    {
+      name: "GMX",
+      symbol: "GMX",
+      address: "",
+      decimals: 18,
+      imageUrl: "https://assets.coingecko.com/coins/images/18323/small/arbit.png?1631532468",
+      isPlatformToken: true,
+    },
+    {
+      name: "Escrowed GMX",
+      symbol: "esGMX",
+      address: "",
+      decimals: 18,
+      isPlatformToken: true,
+    },
+    {
+      name: "GMX LP",
+      symbol: "GLP",
+      address: "",
```

Comment:
We discussed that tokens and markets config should be an external params to SDK, or probably i don't understand  the current concept

Reply:
Those are defaults, it think it's better to keep configs in one place so in future I'm intended to remove duplicates from UI